mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentMapper.java

```diff
@@ -147,15 +156,49 @@ public void storeAttachment(Attachment attachment) throws MailboxException {
         );
     }
 
+    private CompletableFuture<Void> asyncStoreAttachmentForMessageId(Attachment attachment, MessageId ownerMessageId) throws IOException {
```

Comment:
Please rebase on https://github.com/linagora/james-project/pull/960

 => This SHOULD be a DAO with prepared query.

Reply:
ok